assets/model_monitoring/components/src/shared_utilities/df_utils.py

```diff
@@ -170,3 +182,51 @@ def add_value_if_present(
     if row_has_value(row, row_name):
         dict[target_property_name] = row[row_name]
     return dict
+
+
+def try_get_common_columns_with_warning(
```

Comment:
when will the warning and ignore be used?

Reply:
As of right now they are not used but added them for future use if we decide to throw warning in some situations instead of fail job. Or if we don't want to validate for empty common columns in new scenarios.